src/geo/ui/widgets/time-series/torque-histogram-view.js

```diff
@@ -80,6 +80,7 @@ module.exports = View.extend({
 
   _onBrushEnd: function(loBarIndex, hiBarIndex) {
     var data = this.model.getData();
+    this._rangeFilter.set('columnType', this._torqueLayerModel.get('meta').column_type);
```

Comment:
The meta prop is not available initially, right?
I think this detail is not a concern for the view but the model, which knows about both the filter and layer.